Dart/resources/META-INF/plugin.xml

```diff
@@ -103,6 +103,9 @@
     <codeInsight.implementMethod language="Dart"
                                  implementationClass="com.jetbrains.lang.dart.ide.generation.DartImplementMethodHandler"/>
 
+    <statementUpDownMover implementation="com.jetbrains.lang.dart.ide.moveCode.DartStatementMover" id="statement" order="before component" />
```

Comment:
Technology-specific ids are preferred, like DartStatementMover, otherwise someone else (e.g. in 3rd party plugin) can use the same id and plugin loading will fail.